test/functional/wallet_bumpfee.py

```diff
@@ -253,12 +253,13 @@ def test_dust_to_fee(self, rbf_node, dest_address):
     self.log.info('Test that bumped output that is dust is dropped to fee')
     rbfid = spend_one_input(rbf_node, dest_address)
     fulltx = rbf_node.getrawtransaction(rbfid, 1)
-    # size of transaction (p2wpkh, 1 input, 2 outputs): 141 vbytes
-    assert_equal(fulltx["vsize"], 141)
-    # bump with fee_rate of 0.00350000 BTC per 1000 vbytes
-    # expected bump fee of 141 vbytes * fee_rate 0.00350000 BTC / 1000 vbytes = 0.00049350 BTC
+    # expected size of transaction (p2wpkh, 1 input, 2 outputs): 140-141 vbytes
```

Comment:
please explain why the slack(is it the DER encoded sig length difference in witness?)

Reply:
good idea -- done